src/templates/Header.html

```diff
@@ -1,96 +1,99 @@
-<div class="header" id="header">
```

Comment:
So AFAICT the diff looks 🍌 s here but pretty much the only change is the addition of the ref and dynamic `-sticky` class to the top level element, and the stuck header placeholder at the bottom, eh?